docs/user-guide/admin-cli.md

```diff
@@ -9,14 +10,341 @@ by the Docker user via the Docker CLI. However, the Docker CLI is limited in wha
 provide to the user. Furthermore, it is desirable that an administrator be able to get a global view
 of all virtual disks created and in use on the host.  For these reasons, an admin CLI has been
 created that runs on the ESXi host and that provides access to information not visible from the
-Docker CLI. The remainder of this document will describe each admin CLI command and provide examples
-of their usage.
+Docker CLI.
+
+The admin cli also enables ESX admins to implement tenancy.
 
+The remainder of this document will describe each admin CLI command and provide examples
+of their usage.
 
-# ls
+## Tenant
 
 ### Help
+
+```bash
+[root@localhost:~]  /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant -h
+usage: vmdkops_admin.py tenant [-h] {access,rm,create,ls,vm} ...
+
+positional arguments:
+  {access,rm,create,ls,vm}
+    access              Add or remove Datastore access and quotas for a tenant
+    rm                  Delete a tenant
+    create              Create a new tenant
+    ls                  List tenants and the VMs they are applied to
+    vm                  Add, removes and lists VMs in a tenant
+
+optional arguments:
+  -h, --help            show this help message and exit
+```
+### Create
+
+Creates a new named tenant and optionally assigns VMs.
+
+Sample:
+```
+[root@localhost:~]  /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant create --name DevAppX
+tenant create succeeded
+```
+
+The tenant to VM association can be done at create time.
+
+Sample:
+```
+[root@localhost:~]  /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant ls
```

Comment:
Will try to keep the flow as much as possible. But then the commands need to follow a script rather than being independent.